packages/model-viewer/src/features/scene-graph/nodes/node.ts

```diff
@@ -0,0 +1,12 @@
+
+
+export const $children = Symbol('children');
+
+// Defines the base level node methods and data.
+export class Node {
+  name: string = '';
+  [$children] = new Array<Node>();
+  constructor(name: string) {
+    this.name = name;
+  }
+}
```

Comment:
This is too tiny for it's own file; Let's put it in with primitive-node for now; we can re-organize later if needed.

Reply:
Done.